### PR TITLE
Security Guide: replaces support with consulting

### PR DESCRIPTION
**Pull request by Amrita42**
### PR creator: Description

Replace support with consulting for further assistance on the SELinux policy 

### PR creator: Are there any relevant issues/feature requests?

[DOCTEAM-661](https://jira.suse.com/browse/DOCTEAM-661)

### PR creator: Which product versions do the changes apply to?

When opening a PR, check all versions of the documentation that your PR applies to.

- SLE 15/openSUSE Leap 15.x
  - [x] SLE 15 next/openSUSE Leap next *(current `main`, no backport necessary)*
  - [x] SLE 15 SP4/openSUSE Leap 15.5
  - [x] SLE 15 SP4/openSUSE Leap 15.4
  - [x] SLE 15 SP3/openSUSE Leap 15.3
  - [x] SLE 15 SP2/openSUSE Leap 15.2
  - [x] SLE 15 SP1
- SLE 12
  - [ ] SLE 12 SP5
  - [ ] SLE 12 SP4

### PR reviewer only: Have all backports been applied?

The doc team member merging your PR will take care of backporting to older documents.
When opening a PR, do *not* set the following check box.

- [ ] all necessary backports are done

### CODE REVIEW

**Comment by Amrita42** [comment]
@dariavladykina please review thanks

File: xml/selinux.xml [hunk 1]
```diff
@@ -299,9 +299,11 @@ system_u:object_r:var_t var</screen>
...
    The policy is an essential component of &selnx;. &productname; &productnumber;
    does <emphasis>not</emphasis> include a default policy, and you must build a
-   policy that is customized for your installation. &selnx; policies
-   should be customized for your particular needs; consult your &suse;
-   support engineer for assistance.
+   policy that is customized for your installation.
+   &selnx; policies should be customized for your particular needs; consult &suse;
+   consulting services for assistance.
+   We recommend <literal>slemicro</literal> for customers and partners who are looking for a containerized or virtualized
+   host with full &selnx; support including a supported policy.
```
Comment (dariavladykina) [inline xml/selinux.xml:306]:
```suggestion
   host with full &selnx; support, including a supported policy.
```

File: xml/selinux.xml [hunk 2]
```diff
@@ -299,9 +299,11 @@ system_u:object_r:var_t var</screen>
   <para>
    The policy is an essential component of &selnx;. &productname; &productnumber;
    does <emphasis>not</emphasis> include a default policy, and you must build a
-   policy that is customized for your installation. &selnx; policies
-   should be customized for your particular needs; consult your &suse;
-   support engineer for assistance.
+   policy that is customized for your installation.
+   &selnx; policies should be customized for your particular needs; consult &suse;
```
Comment (dariavladykina) [inline xml/selinux.xml:303]:
```suggestion
   &selnx; policies should be customized for your particular needs. Contact &suse;
```

**Review by dariavladykina — COMMENTED** [review]
Hi, please see some suggestions here. Thanks!

**Comment by Amrita42** [comment]
Thanks for the quick review @dariavladykina

**Comment by Amrita42** [comment]
Merged to all the SLE15 SP versions

**Pull request merged**